### PR TITLE
Qt6 Windows tweaks

### PR DESCRIPTION
**Pull request by psieg**

### CODE REVIEW

File: Software/src/UpdatesProcessor.cpp [hunk 1]
```diff
@@ -265,35 +265,35 @@ bool UpdatesProcessor::isVersionMatches(const QString &predicate, const AppVersi
 QList<UpdateInfo> * UpdatesProcessor::readUpdates(QList<UpdateInfo> *updates, QXmlStreamReader *xmlReader)
 {
 	if (xmlReader->readNextStartElement()) {
-		while (xmlReader->name() == "update") {
+		while (xmlReader->name() == QString("update")) {
```
Comment (zomfg) [inline Software/src/UpdatesProcessor.cpp:268]: would better with `QStringLiteral` https://github.com/KDE/clazy/blob/master/docs/checks/README-qstring-allocations.md

Comment (psieg) [reply Software/src/UpdatesProcessor.cpp]: Thanks

File: Software/src/UpdatesProcessor.cpp [hunk 2]
```diff
@@ -78,7 +78,7 @@ QList<UpdateInfo> UpdatesProcessor::readUpdates()
 	QXmlStreamReader xmlReader(_reply->readAll());
 
 	if (xmlReader.readNextStartElement()) {
-		if (xmlReader.name() == "updates") {
+		if (xmlReader.name() == QString("updates")) {
```
Comment (zomfg) [inline Software/src/UpdatesProcessor.cpp:81]: missed one

Comment (psieg) [reply Software/src/UpdatesProcessor.cpp]: Damn, here I go force pushing again

**Pull request merged**